index.html

```diff
@@ -346,14 +360,52 @@ <h3 class="page-heading">What is Hackcouver?</h3>
 
                     <div class="col-xl-5 col-md-5">
                         <div class="man_thumb">
-                            <img src="img/ilstrator/man.png" alt="" />
+                            <img src="img/team_photo-removebg-preview.png" alt="" />
                         </div>
                     </div>
                 </div>
             </div>
         </div>
         <!--/ compapy_info  -->
 
+        <div class="meet-our-team mx-3 px-3">
+            <div class="container text-center no-border">
+                <div class="row">
+                    <div class="col-4">
+                        <div class="card">
+                            <img src="..." class="card-img-top" alt="...">
+                            <div class="card-body">
+                                <h5 class="card-title">Sam Mitchell</h5>
+                                <h5 class="card-subtitle">Human Resources</h5>
+                                <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
```

Comment:
Still WIP? Are we adding the rest of the team as well?

Reply:
Yes this was just a template